i/importlib_resources/importlib_resources_ubi_8.4.sh

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : importlib_resources
+# Version       : v5.2.2, v5.2.1
```

Comment:
I see that requested version is 5.1.2, can it not be tested ?

Reply:
The build and tests passed on V5.1.2 and the results as below.

[root@eeccef237779 ~]# sh importlib_resources_ubi_8.4.sh v5.1.2
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:02:15 ago on Wed Sep 22 07:18:41 2021.
Complete!
Installing collected packages: six, filelock, distlib, platformdirs, zipp, importlib-resources, typing-extensions, importlib-metadata, backports.entry-points-selectable, virtualenv, py, toml, pyparsing, packaging, pluggy, tox
Successfully installed backports.entry-points-selectable-1.1.0 distlib-0.3.3 filelock-3.0.12 importlib-metadata-4.8.1 importlib-resources-5.2.2 packaging-21.0 platformdirs-2.3.0 pluggy-1.0.0 py-1.10.0 pyparsing-2.4.7 six-1.16.0 toml-0.10.2 tox-3.24.4 typing-extensions-3.10.0.2 virtualenv-20.8.0 zipp-3.5.0
Cloning into 'importlib_resources'...
remote: Enumerating objects: 2843, done.
remote: Counting objects: 100% (121/121), done.
remote: Compressing objects: 100% (42/42), done.
remote: Total 2843 (delta 94), reused 85 (delta 79), pack-reused 2722
Receiving objects: 100% (2843/2843), 420.42 KiB | 1.74 MiB/s, done.
Resolving deltas: 100% (1843/1843), done.
Note: switching to 'v5.1.2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at d094cac Update changelog.
 found to checkout
py36 create: /root/importlib_resources/.tox/py36
py36 develop-inst: /root/importlib_resources
py36 installed: attrs==21.2.0,black==21.9b0,click==8.0.1,coverage==5.5,dataclasses==0.8,docutils==0.17.1,filelock==3.0.12,flake8==3.9.2,importlib-metadata==4.8.1,-e git+https://github.com/python/importlib_resources@d094cac95634795704b710f97716a72d0599747a#egg=importlib_resources,iniconfig==1.1.1,jaraco.context==4.0.0,jaraco.functools==3.3.0,mccabe==0.6.1,more-itertools==8.10.0,mypy==0.910,mypy-extensions==0.4.3,packaging==21.0,pathspec==0.9.0,pep517==0.11.0,platformdirs==2.3.0,pluggy==1.0.0,py==1.10.0,pycodestyle==2.7.0,pyflakes==2.3.1,pyparsing==2.4.7,pytest==6.2.5,pytest-black==0.3.12,pytest-checkdocs==2.7.1,pytest-cov==2.12.1,pytest-enabler==1.2.0,pytest-flake8==1.0.7,pytest-mypy==0.8.1,regex==2021.8.28,toml==0.10.2,tomli==1.2.1,typed-ast==1.4.3,typing-extensions==3.10.0.2,zipp==3.5.0
py36 run-test-pre: PYTHONHASHSEED='1419529250'
py36 run-test: commands[0] | pytest
==================================================================== test session starts ====================================================================
platform linux -- Python 3.6.8, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
cachedir: .tox/py36/.pytest_cache
rootdir: /root/importlib_resources, configfile: pytest.ini
plugins: checkdocs-2.7.1, mypy-0.8.1, cov-2.12.1, flake8-1.0.7, enabler-1.2.0, black-0.3.12
collected 226 items

. .                                                                                                                                                   [  0%]
setup.py ....                                                                                                                                         [  2%]
docs/conf.py ...                                                                                                                                      [  3%]
importlib_resources/__init__.py ...                                                                                                                   [  4%]
importlib_resources/_adapters.py ...                                                                                                                  [  6%]
importlib_resources/_common.py ...                                                                                                                    [  7%]
importlib_resources/_compat.py ...                                                                                                                    [  8%]
importlib_resources/_py3.py ...                                                                                                                       [ 10%]
importlib_resources/abc.py ...                                                                                                                        [ 11%]
importlib_resources/readers.py ...                                                                                                                    [ 12%]
importlib_resources/simple.py ...                                                                                                                     [ 14%]
importlib_resources/tests/__init__.py ...                                                                                                             [ 15%]
importlib_resources/tests/_compat.py ...                                                                                                              [ 16%]
importlib_resources/tests/test_files.py .....s..s                                                                                                     [ 20%]
importlib_resources/tests/test_open.py .............................................                                                                  [ 41%]
importlib_resources/tests/test_path.py ....................                                                                                           [ 50%]
importlib_resources/tests/test_read.py .....................................                                                                          [ 66%]
importlib_resources/tests/test_reader.py ...............                                                                                              [ 73%]
importlib_resources/tests/test_resource.py .........................s......                                                                           [ 87%]
importlib_resources/tests/update-zips.py ....                                                                                                         [ 89%]
importlib_resources/tests/util.py ...                                                                                                                 [ 90%]
importlib_resources/tests/data01/__init__.py ...                                                                                                      [ 91%]
importlib_resources/tests/data01/subdirectory/__init__.py ...                                                                                         [ 93%]
importlib_resources/tests/data02/__init__.py ...                                                                                                      [ 94%]
importlib_resources/tests/data02/one/__init__.py ...                                                                                                  [ 95%]
importlib_resources/tests/data02/two/__init__.py ...                                                                                                  [ 97%]
importlib_resources/tests/zipdata01/__init__.py ...                                                                                                   [ 98%]
importlib_resources/tests/zipdata02/__init__.py ...                                                                                                   [100%]

===================================================================== warnings summary ======================================================================
setup.py: 2 warnings
docs/conf.py: 2 warnings
importlib_resources/__init__.py: 2 warnings
importlib_resources/_adapters.py: 2 warnings
importlib_resources/_common.py: 2 warnings
importlib_resources/_compat.py: 2 warnings
importlib_resources/_py3.py: 2 warnings
importlib_resources/abc.py: 2 warnings
importlib_resources/readers.py: 2 warnings
importlib_resources/simple.py: 2 warnings
importlib_resources/tests/__init__.py: 2 warnings
importlib_resources/tests/_compat.py: 2 warnings
importlib_resources/tests/test_files.py: 2 warnings
importlib_resources/tests/test_open.py: 2 warnings
importlib_resources/tests/test_path.py: 2 warnings
importlib_resources/tests/test_read.py: 2 warnings
importlib_resources/tests/test_reader.py: 2 warnings
importlib_resources/tests/test_resource.py: 2 warnings
importlib_resources/tests/update-zips.py: 2 warnings
importlib_resources/tests/util.py: 2 warnings
importlib_resources/tests/data01/__init__.py: 2 warnings
importlib_resources/tests/data01/subdirectory/__init__.py: 2 warnings
importlib_resources/tests/data02/__init__.py: 2 warnings
importlib_resources/tests/data02/one/__init__.py: 2 warnings
importlib_resources/tests/data02/two/__init__.py: 2 warnings
importlib_resources/tests/zipdata01/__init__.py: 2 warnings
importlib_resources/tests/zipdata02/__init__.py: 2 warnings
  /root/importlib_resources/.tox/py36/lib/python3.6/site-packages/flake8/plugins/manager.py:254: DeprecationWarning: SelectableGroups dict interface is deprecated. Use select.
    eps = importlib_metadata.entry_points().get(self.namespace, ())

-- Docs: https://docs.pytest.org/en/stable/warnings.html

----------- coverage: platform linux, python 3.6.8-final-0 -----------
Name                                                                                       Stmts   Miss  Cover   Missing
------------------------------------------------------------------------------------------------------------------------
docs/conf.py                                                                                   3      0   100%
importlib_resources/__init__.py                                                                4      0   100%
importlib_resources/_adapters.py                                                              40     12    70%   25, 28, 39, 47, 50, 53, 60-61, 67-68, 71, 82
importlib_resources/_common.py                                                                53      0   100%
importlib_resources/_compat.py                                                                43      0   100%
importlib_resources/_py3.py                                                                   82      3    96%   33-34, 168
importlib_resources/abc.py                                                                    46     13    72%   20, 33, 41, 46, 66-67, 73-74, 98, 128, 131, 134, 137
importlib_resources/readers.py                                                                73      8    89%   23, 36-39, 44-45, 102
importlib_resources/simple.py                                                                 45     19    58%   50, 60-61, 64, 67, 70-73, 76, 86, 89, 92, 95-97, 100, 103, 116
importlib_resources/tests/__init__.py                                                          0      0   100%
importlib_resources/tests/_compat.py                                                          12      0   100%
importlib_resources/tests/data01/__init__.py                                                   0      0   100%
importlib_resources/tests/data01/subdirectory/__init__.py                                      0      0   100%
importlib_resources/tests/data02/__init__.py                                                   0      0   100%
importlib_resources/tests/data02/one/__init__.py                                               0      0   100%
importlib_resources/tests/data02/two/__init__.py                                               0      0   100%
importlib_resources/tests/test_files.py                                                       24      2    92%   26, 39
importlib_resources/tests/test_open.py                                                        46      1    98%   77
importlib_resources/tests/test_path.py                                                        34      1    97%   61
importlib_resources/tests/test_read.py                                                        37      1    97%   60
importlib_resources/tests/test_reader.py                                                      77      1    99%   123
importlib_resources/tests/test_resource.py                                                   144     10    93%   163-164, 169-170, 174-176, 195-197, 254
importlib_resources/tests/update-zips.py                                                      24      0   100%
importlib_resources/tests/util.py                                                            121      9    93%   41, 50, 67, 149-150, 155-156, 161-162
importlib_resources/tests/zipdata01/__init__.py                                                0      0   100%
importlib_resources/tests/zipdata01/ziptestdata.zip/ziptestdata/__init__.py                    0      0   100%
importlib_resources/tests/zipdata01/ziptestdata.zip/ziptestdata/subdirectory/__init__.py       0      0   100%
importlib_resources/tests/zipdata02/__init__.py                                                0      0   100%
importlib_resources/tests/zipdata02/ziptestdata.zip/ziptestdata/__init__.py                    0      0   100%
importlib_resources/tests/zipdata02/ziptestdata.zip/ziptestdata/one/__init__.py                0      0   100%
importlib_resources/tests/zipdata02/ziptestdata.zip/ziptestdata/two/__init__.py                0      0   100%
setup.py                                                                                       3      0   100%
------------------------------------------------------------------------------------------------------------------------
TOTAL                                                                                        911     80    91%

=========================================================================== mypy ============================================================================
Success: no issues found in 27 source files
================================================== 221 passed, 3 skipped, 54 warnings in 74.66s (0:01:14) ===================================================
__________________________________________________________________________ summary __________________________________________________________________________
  py36: commands succeeded
  congratulations :)
Build & Test Success for python 3.6 environment
[root@eeccef237779 ~]#

Reply:
Ok.. Pl. update this version also in the header